src/components/event-details/EventDetails.tsx

```diff
@@ -52,15 +55,24 @@ const Info = styled.div`
 `;
 
 const AnnouncementBox = styled.div`
-  align-items: center;
-  border: 1px solid #d9d9d9;
-  min-height: 80px;
+  padding: 15px;
+`;
+
+const AnnouncementNoContent = styled(AnnouncementBox)`
+  border: 1px solid black;
+  text-align: center;
 `;
 
-const EventListing: React.FC<EventProps> = ({ thumbnail, title, details }) => {
-  const defaultImg =
-    'https://lucys-love-bus-public.s3.us-east-2.amazonaws.com/LLB_2019_Sq_rgb+1.png';
+export interface EventListingProps extends EventProps {
+  announcements?: EventAnnouncement[];
+}
 
+const EventListing: React.FC<EventListingProps> = ({
```

Comment:
📝 Jennifer and I changed this component's name to be EventDetails. Don't know how that went undetected for so long

Reply:
❗ Yeah that might be a problem, not sure how merging will work here